CHANGELOG.md

```diff
@@ -1,3 +1,431 @@
+## v0.3.0 - 2020-07-29
+
+We're happy to announce the release of Lokomotive v0.3.0 (Coast Starlight).
+
+This release packs new features and bugfixes. Some of the highlights are:
+
+* Kubernetes 1.18.6
+* AKS Kubernetes 1.16.10
```

Comment:
This seems confusing. Maybe something like:

```suggestion
* Kubernetes 1.18.6
* For Lokomotive clusters running on top of AKS, Kubernetes 1.16.10 is now used
```

Reply:
```suggestion
* Kubernetes 1.18.6
* For Lokomotive clusters running on top of AKS, Kubernetes 1.16.10 is installed.
```

How does that sound ?

Reply:
Perfect! :)